### DISCUSSION TITLE
lost coins

### DISCUSSION THREAD

**Discussion by gradkv** [Chia-Network/chia-blockchain#13036] [Support]
After upgrading from 1.3.5 to 1.5, the coins moved from my old address to another address.  And there is no information in the wallet. How to access them

**Comment by gradkv** [comment]
old address
xch1mvk3x6hthjxjyg238zrg4a78qw2820fvvxhjwrhtqaxhph5qh89q4gpjh8
![image](https://user-images.githubusercontent.com/111477617/185286793-c7977fd4-7f3f-4cd8-98fb-741038f6388a.png)

**Reply by Nightbeem** [reply]
Hi.  I had a similar problem and obtained this solution from the Chia Keybase support chat:

_If you're on a machine running a full node, open the config.yaml, under the wallet section set `connect_to_unknown_peers: false` (may need to add this line), then delete your wallet db at `.chia/mainnet/wallet/db` (1.5 should have created a new one with `_r1_` in the filename), then start chia. this will force the wallet to resync only off of your own full node, which should prevent missing coin records._

Doing this solved the problem.

**Reply by gradkv** [reply]
Did it:
If you're on a machine running a full node, open the config.yaml, under the wallet section set connect_to_unknown_peers: false (may need to add this line), then delete your wallet db at .chia/mainnet/wallet/db (1.5 should have created a new one with _r1_ in the filename), then start chia. this will force the wallet to resync only off of your own full node, which should prevent missing coin records.
the transaction history appeared, the coins move to an unknown address.
I don't know what this is related to
